fleetctl/fleetctl.go

```diff
@@ -482,6 +482,42 @@ func getChecker() *ssh.HostKeyChecker {
 	return ssh.NewHostKeyChecker(keyFile)
 }
 
+func getUnitFile(file string) (*unit.UnitFile, error) {
```

Comment:
Please add a docstring clarifying when exactly this should be used: in particular, after the registry has already been checked for a unit of the given name. This is to mitigate my comment @ https://github.com/coreos/fleet/pull/1433#discussion_r53453416 - i.e. that this function can return an error which references the registry.